### ISSUE TITLE
Definitions in the guidance repeated, not referenced  

### ISSUE THREAD

**Issue by JachymHercher** [open-contracting/standard#1181]
In https://standard.open-contracting.org/latest/en/guidance/map/awards_contracts_buyers_suppliers/ (and possibly elsewhere), definitions of fields (award, contract, buyer, etc.) are included in a way that doesn't update automatically when the definition changes in the schema. This breaks the "single source of truth" approach, introducing risk of divergence between the schema and the guidance. 

Is there a way to include the definitions in a self-updating way? If yes, we should go through the guidance and change any definitions to references (possibly once the guidance is stable for version 1.2).

**Comment by jpmckinney** [comment]
I believe we can author a Sphinx directive to do this, and it should be straightforward. Tasks:

* [x] Identify locations of definitions (we can perhaps start with some automated approach)
* [x] Author Sphinx directive
* [x] Apply Sphinx directive

This can be done against 1.1-dev, and then merged into 1.2-dev, instead of waiting for a final 1.2.

**Comment by jpmckinney** [comment]
We fixed this in #1204. The new directives are documented at https://sphinxcontrib-opencontracting.readthedocs.io/en/latest/

For "Identify locations of definitions", I don't think we did a complete audit. Looking at the locations where lines start with `>` (i.e. a blockquote) and where we use constructions like `define.* as` or `describe.* as\b`, we have:

* guidance/map/contracting_planning_processes.md: This contains the definitions of contracting process and planning process, which are contained within the top-level `description` field of `release-schema.json`.
* primer/how.md: Reuses the first sentence for the definition of contracting process.

We also had an old note in `guidance/map/buyers_suppliers.md` that should have been removed with #1163.

I've opened #1566 to add comments to the two locations and remove the note.
